<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_Book</class>
 <widget class="QDialog" name="search_Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1826</width>
    <height>1218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Book</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/logo.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>420</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:4px;</string>
   </property>
   <property name="text">
    <string>Search Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Search By</string>
   </property>
  </widget>
  <widget class="QComboBox" name="search_by">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:2px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_box">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>300</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Search Box</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Details_table">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>701</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Book ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Book Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>publication</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edition</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Author Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
